src/components/Select/Select.js

```diff
@@ -24,11 +25,11 @@ class Select extends React.Component {
     name: PropTypes.string,
     options: PropTypes.arrayOf(PropTypes.object),
     width: PropTypes.number
-  }
+  };
 
   componentDidMount = () => {
-    bindUIEvents(this.selectRef.current)
-  }
+    requiredSelect()
```

Comment:
This doesn't seem equivalent

Reply:
the new file to import select functions from is here:
import { requiredSelect } from '@sparkdesignsystem/spark-core/base/requiredSelect'

and there is no longer a bindUIEvents function being exported, only the main requiredSelect function that calls the bindUIEvents function under the hood